resources/definitions/creality_sermoond1.def.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "Creality Sermoon D1",
+    "version": 2,
+    "inherits": "creality_base",
+    "overrides": {
+        "machine_name": { "default_value": "Creality Sermoon D1" },
+        "machine_width": { "default_value": 280 },
+        "machine_depth": { "default_value": 260 },
+        "machine_height": { "default_value": 310 },
+        "machine_head_polygon": { "default_value": [

```

Comment:
This setting doesn't exist (any more). That's what the automated tests are complaining about. I think you meant to just override `machine_head_with_fans_polygon` instead?

Reply:
Thanks for the information. I created the configurations with the template of the Creality Slicer. However, this one was a different version. I hope this is correct now. :)